lib/inMemFileDB.js

```diff
@@ -69,7 +69,9 @@ class InMemoryFileDB {
 
         // path is always relative to appName.js-controller
         this.dataDir = (this.settings.connection.dataDir || getDefaultDataDir());
-        this.dataDir = path.normalize(path.join(__dirname, '..', this.dataDir) + path.sep);
+        if (this.dataDir.startsWith('.')) {
```

Comment:
Does this check for relative vs absolute paths? Then the check should be `if (!path.isAbsolute(this.dataDir))`. And the comment above is wrong because it states that the path is always relative.

Reply:
I hd the case on travis where it was anabsolute path and so it broke and original code checked for .. or ./ on the beginning :-) so es comment is wrong

Reply:
see https://github.com/ioBroker/ioBroker.js-controller/blob/master/lib/objects/objectsInMemServer.js#L44 ... original code. I changed it and removed the comment